receiver/sapnetweaverreceiver/README.md

```diff
@@ -0,0 +1,34 @@
+# SAP Netweaver Receiver
```

Comment:
I think these docs miss a few things from our new standards, I'll forward you the standards doc on slack.

Reply:
Updated to documentation